src/store/migrations.js

```diff
@@ -17,15 +17,34 @@
 
 import migrate from 'redux-storage-decorator-migrate';
 
+// Update this constant whenever the Redux reducers change in a
+// not-backwards-incompatible way:
 const STATE_VERSION = 1;
 
+export function migrationFromZeroToOne(state: any) {
```

Comment:
IMO this should be called `migrateToReduxStorage` with a JSDoc indicating that migrates from zero to one, since that more clearly specifies what happens. Similarly, in the future, if we add a new key, the migration would be called `addFooKey` with JSDoc specifying that it migrates from one to two. What do you think?

Reply:
+1

That makes it very clear what the migration is in place for. Honestly, `handleMigrations` makes it clear what order the migrations are to be applied, so I don't even think the JSDoc comment is necessary.

Reply:
Yeah, I suppose my only reluctance is some migrations might do a lot of little things and be hard to summarize neatly, it's one other thing you have to come up with a name for. But that's a pretty weak argument. I'll change it.